src/ResourceManager/RecoveryServices/Commands.RecoveryServices/Vault/GetAzureRmRecoveryServicesVaults.cs

```diff
@@ -25,7 +25,7 @@ namespace Microsoft.Azure.Commands.RecoveryServices
     /// Retrieves Azure Recovery Services Vault.
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "AzureRmRecoveryServicesVault")]
-    [OutputType(typeof(List<ARSVault>))]
+    [OutputType(typeof(ARSVault))]
```

Comment:
@twitchax same comment about the [output for the markdown](https://github.com/twitchax/azure-powershell/blob/8fb2d3451cd510ff7e2018fdf1d2bd3d84f83225/src/ResourceManager/RecoveryServices/Commands.RecoveryServices/help/Get-AzureRmRecoveryServicesVault.md#outputs) of this cmdlet not being updated